{"name": "编辑用户手机号为异常号码", "status": "passed", "description": "编辑用户手机号为异常号码", "attachments": [{"name": "log", "source": "a1dbdde6-1d22-4434-8d1d-31465337f300-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '编辑用户手机号为异常号码', 'nickname': '刘子编辑', 'phone': 1595485421655, 'real_name': '刘子编辑', 'identities': 5, 'gender': 2, 'stu_no': 9658}"}], "start": 1723083903746, "stop": 1723083903907, "uuid": "10838b01-35b1-458b-b93e-872dcc5c03b7", "historyId": "815277452944b81bf4e3cb4b51261a0f", "testCaseId": "0353a34ef4599ada0d9df8f9de31b672", "fullName": "testcases.test_user_manage.TestUserManageApi#test_edit_user", "labels": [{"name": "story", "value": "编辑用户手机号为异常号码测试用例"}, {"name": "feature", "value": "用户管理模块"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_manage"}, {"name": "subSuite", "value": "TestUserManageApi"}, {"name": "host", "value": "常胜峰"}, {"name": "thread", "value": "1232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_manage"}]}